src/transformers/testing_utils.py

```diff
@@ -654,21 +655,42 @@ def get_env(self):
         env["PYTHONPATH"] = ":".join(paths)
         return env
 
-    def get_auto_remove_tmp_dir(self, tmp_dir=None, after=True, before=False):
+    def get_auto_remove_tmp_dir(self, tmp_dir=None, before=None, after=None):
         """
         Args:
             tmp_dir (:obj:`string`, `optional`):
-                use this path, if None a unique path will be assigned
-            before (:obj:`bool`, `optional`, defaults to :obj:`False`):
-                if `True` and tmp dir already exists make sure to empty it right away
-            after (:obj:`bool`, `optional`, defaults to :obj:`True`):
-                delete the tmp dir at the end of the test
+                if :obj:`None`:
+
+                   - a unique tmp path will be created
```

Comment:
Suggestion to avoid making a list:
```
If unset, a unique tmp path will be created, :obj:`before` and :obj:`after` will default to :obj:`True`. Otherwise a unique tmp path will be chosen and created, :obj:`before` will default to :obj:`True` and :obj:`after` will default to :obj:`False`.
```
I copied the beginning of the "else" part (a unique tmp path will be chosen and created) but I don't understand it: how is the argument used in this case? Also abbreviations like `tmp path` should be avoided in the documentayion, we have space to use proper English :-)

Reply:
That works, but would you agree that the proposed rewrite is much more difficult to grasp? Making docs more difficult to understand because of formatters is strange to me. I'd rather keep the weird new line. 

It's not a list btw, should I make it a list and add bullets if we choose to keep it that way?

In my experience if you live in a unix world `tmp` is sort of a term and it's heavily used in the tech documentation, and therefore it's much quicker to grasp than "temporary" because our brains have been trained on it - but I have no problem expanding it.

Reply:
Thank you very much for the improvements you have been making, @sgugger!